cypress/e2e/models/administration/custom-migration-targets/custom-migration-target.ts

```diff
@@ -56,6 +64,7 @@ export class CustomMigrationTarget {
     }
 
     public create() {
+        CustomMigrationTarget.selectLanguage(this.language);
```

Comment:
selectLanguage will do CustomMigrationTarget.open(); and openNewForm is also doing CustomMigrationTarget.open();

Wouldn't that reload the page and language selection will be lost ?

Reply:
No, because I'm not forcing a reload, so it will check if its already in the URL and because it its, it won't reload the page.